vis/notebook/vis.ipynb

```diff
@@ -49,10 +49,12 @@
       "metadata": {},
       "outputs": [],
       "source": [
+        "import os\n",
+        "NOMIC_TONY_API_TOKEN = os.environ['NOMIC_TONY_API_TOKEN']\n",
         "# Run this if you don't have a nomic access token\n",
         "#!nomic login\n",
         "# Run this if you have nomic access token\n",
-        "!nomic login 'HfFeBu516-CFtRnOFlEY9k-A5DWMqyKrhNe7XyxSv3irV'\n",
```

Comment:
the key is still in the history.

1) rebase to squash your changes into a single commit.
2) make sure you've invalidated this key so it doesn't work anymore (as you have to assume someone scraped it)